.github/SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Support
+
+Terraform is a mature project with a growing community. There are active, dedicated people willing to help you through various mediums.
+
+Take a look at those mediums listed at https://www.terraform.io/community.html
```

Comment:
Thinking whether we need this file, or whether it should be named differently. If you're using provider and read this file it's not really what you'd expect to find in it, as the content is for Terraform as a tool not specific provider.

CC: @Didainius

Reply:
We could also just name it TERRAFORM.md to reflect the contents.

Reply:
I vote removing too. Reading the contents, doesn't really help much, tbh.

Reply:
Let's remove. I don't see issue. We can add if it is needed

Reply:
Done